inc/saivlan.h

```diff
@@ -386,6 +386,19 @@ typedef enum _sai_vlan_attr_t
      */
     SAI_VLAN_ATTR_BROADCAST_FLOOD_GROUP,
 
+    /**
+     * @brief Enable/Disable ARP & NDP suppression
+     *
+     * When enabled, the ARP and NDP messages are trapped to CPU instead of
+     * getting copied (and flooding the vlan). Default is set to false
+     * for backward compatibility reasons
+     *
+     * @type bool
+     * @flags CREATE_AND_SET
+     * @default false
+     */
+    SAI_VLAN_ATTR_NEIGHBOR_SUPPRESSION_ENABLE,
```

Comment:
maybe an attribute SAi_VLAN_ATTR_TRAP_DISABLE_LIST, which is a trap object list.

Reply:
the behavior can be achieved by creating a trap of type 
`
    /** Default packet action is forward */
    SAI_HOSTIF_TRAP_TYPE_ARP_REQUEST = 0x00002000,

    /** Default packet action is forward */
    SAI_HOSTIF_TRAP_TYPE_ARP_RESPONSE = 0x00002001,
`
and then add the entry with type 
`
    /** Vlan based Host Interface entry Type */
    SAI_HOSTIF_TABLE_ENTRY_TYPE_VLAN,
`
and define the action on the entry.